src/pages/discord/link/components/view/index.tsx

```diff
@@ -0,0 +1,89 @@
+import { useWallet } from "@solana/wallet-adapter-react";
```

Comment:
It seems github see the `view` folder small, trying to delete it and create it again maybe will fix the issue